measurements/pages/__init__.py

```diff
@@ -220,12 +220,13 @@ def decompress_autoclaved(
     def generator():
         try:
             url = urljoin(current_app.config['AUTOCLAVED_BASE_URL'], autoclaved_filename)
-            headers = {"Range": "bytes={}-{}".format(frame_off, frame_off + total_frame_size)}
+            # byte positions specified are inclusive -- https://tools.ietf.org/html/rfc7233#section-2.1
+            headers = {"Range": "bytes={}-{}".format(frame_off, frame_off + total_frame_size - 1)}
             r = requests.get(url, headers=headers, stream=True)
             beginning = True
             # Create a copy because we are in a closure
             to_read = report_size
-            while to_read > 0:
+            while to_read > 1:
```

Comment:
This is `1` instead of `0` because `report_size` is actually include of the newline character?

Reply:
Yep. And the last `\n` has no information regarding frame if it's separate one.